src/constants.ts

```diff
@@ -12,3 +13,15 @@ export const SUPPORTED_EXTENSION_IDS = [
 ];
 
 export const SIMPLE_DECISIONS_ACTION_CODE = '0x12345678';
+
+const toB32 = (input: BigNumberish): string =>
```

Comment:
I remember we had a util for this somewhere, but for the life of me can't find it right now